dask_ml/model_selection/_incremental.py

```diff
@@ -106,6 +109,12 @@ def _create_model(model, ident, **params):
         return model, {"model_id": ident, "params": params, "partial_fit_calls": 0}
 
 
+def _show_msg(msg, verbose):
```

Comment:
Instead of this, I'd prefer to just use the standard logging module for this. Users can get the info printed to stdout by adding their own handler.

We could consider doing that for them if they make an estimator with `verbose=True`.

```python
handler = logging.StreamHandler()
handler.setLevel(logging.INFO)

logger.addHandler(handler)
```

It may be good to get the log formatter from distributed and apply that.

We'd need to make sure to only add the handler once (maybe the logging module does that already).